This patch Removes pointer to old printer driver (gedevmd2k.c) that only includes 
references to use under the Aladdin Free Public License (which does not allow 
for commercial distribution) from Ghostscript.
Bug # 15712747

diff --git a/configure.ac b/configure.ac
index a234d2f..66f6562 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2411,7 +2411,7 @@ APPLE_DEVS='appledmp iwhi iwlo iwlq'
 IBM_DEVS='ibmpro jetp3852'
 OKI_DEVS='oki182 okiibm oki4w'
 JAPAN_DEVS='lips4 lips4v ljet4pjl lj4dithp dj505j picty180 lips2p bjc880j pr201 pr150 pr1000 pr1000_4 jj100 bj10v bj10vh mj700v2c mj500c mj6000c mj8000c fmpr fmlbp ml600 lbp310 lbp320 md50Mono md50Eco md1xMono escpage lp2000 npdl rpdl'
-MISC_PDEVS='uniprint ap3250 atx23 atx24 atx38 itk24i itk38 coslw2p coslwxl declj250 fs600 imagen lj250 m8510 necp6 oce9050 r4081 sj48 tek4696 t4693d2 t4693d4 t4693d8 dl2100 la50 la70 la75 la75plus ln03 xes md2k md5k gdi samsunggdi'
+MISC_PDEVS='uniprint ap3250 atx23 atx24 atx38 itk24i itk38 coslw2p coslwxl declj250 fs600 imagen lj250 m8510 necp6 oce9050 r4081 sj48 tek4696 t4693d2 t4693d4 t4693d8 dl2100 la50 la70 la75 la75plus ln03 xes gdi samsunggdi'
 OPVP_DEVS='opvp oprp'
 
 ETS_HALFTONING_DEVS='rinkj'
diff --git a/contrib/contrib.mak b/contrib/contrib.mak
index bb26640..f9fdaac 100644
--- a/contrib/contrib.mak
+++ b/contrib/contrib.mak
@@ -676,23 +676,6 @@ $(DD)bjc880j.dev : $(lipsr_) $(DD)page.dev \
 	$(SETPDEV) $(DD)bjc880j $(lipsr_)
 
 
-### ------------ ALPS MD series printer devices ------------ ###
-
-md2k_=$(DEVOBJ)gdevmd2k.$(OBJ)
-
-$(DD)md2k.dev : $(md2k_) $(DD)page.dev \
-                           $(CONTRIB_MAK) $(MAKEDIRS)
-	$(SETPDEV) $(DD)md2k $(md2k_)
-
-$(DD)md5k.dev : $(md2k_) $(DD)page.dev \
-                           $(CONTRIB_MAK) $(MAKEDIRS)
-	$(SETPDEV) $(DD)md5k $(md2k_)
-
-$(DEVOBJ)gdevmd2k.$(OBJ) : $(CONTRIBSRC)gdevmd2k.c $(PDEVH) $(gsparam_h) \
-                           $(CONTRIB_MAK) $(MAKEDIRS)
-	$(DEVCC) $(DEVO_)gdevmd2k.$(OBJ) $(C_) $(CONTRIBSRC)gdevmd2k.c
- 
-
 ### ----------------- The Okidata OkiPage 4w+ device ------------------- ###
 
 oki4w_=$(DEVOBJ)gdevop4w.$(OBJ)
